src/ExEss/Cms/Controller/User/CurrentController.php

```diff
@@ -0,0 +1,35 @@
+<?php declare(strict_types=1);
+
+namespace ExEss\Cms\Controller\User;
+
+use ExEss\Cms\Service\UserService;
+use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
+use Symfony\Component\HttpFoundation\JsonResponse;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\Routing\Annotation\Route;
+use Symfony\Component\Security\Core\Authentication\Token\Storage\TokenStorageInterface;
+
+class CurrentController extends AbstractController
+{
+    private TokenStorageInterface $tokenStorage;
+
+    private UserService $userService;
+
+    public function __construct(TokenStorageInterface $tokenStorage, UserService $userService)
+    {
+        $this->tokenStorage = $tokenStorage;
```

Comment:
might be better to still use the src/ExEss/Cms/Api/V8_Custom/Service/Security.php service here, it checks is there is a user (which for this route should always be the case, since it's protected, so if null you can just throw an error, and no need to do the magic with the token storage

Reply:
done